rules-tests/Php81/Rector/ClassMethod/NewInInitializerRector/Fixture/skip_if_no_default_null.php.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Rector\Tests\Php81\Rector\ClassMethod\NewInInitializerRector\Fixture;
+
+use Php81\Rector\ClassMethod\NewInInitializerRector\Source\Logger\DefaultLogger;
+use Php81\Rector\ClassMethod\NewInInitializerRector\Source\Logger\NullLogger;
+
+final class SkipIfNoDefaultNull
+{
+    private DefaultLogger $logger;
+
+    public function __construct(
+        ?DefaultLogger $logger,
+    ) {
+        $this->logger = $logger ?? new NullLogger();
+    }
+}
```

Comment:
@samsonasik This is one of cases that would always throw error if upgraded.

Here, without `null` default, the `null` is always passed from outside to trigger:

```php
new SkipIfNoDefaultNull(null); // only way to create NullLogger
```